network/p2p/unicast_manager.go

```diff
@@ -0,0 +1,17 @@
+package p2p
+
+import (
+	"context"
+
+	libp2pnet "github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/multiformats/go-multiaddr"
+
+	"github.com/onflow/flow-go/network/p2p/unicast/protocols"
+)
+
+type UnicastManager interface {
```

Comment:
Please add godoc for the interface and its methods.

Reply:
https://github.com/onflow/flow-go/pull/3902/commits/a4f7a3721c53618b0cb54633a885fa01da34a00c